### PR TITLE
Hxietest chicago request from hill

### PR DESCRIPTION
**Pull request by hxietest**
please review

### CODE REVIEW

File: hxietest.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Chicago
+- Place 1 :item1: & :item1_1
```
Comment (yangineer) [inline hxietest.md:2]: What are item1 and item1_1?
Please be more specific.

File: hxietest.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+- Place 2 :item2
```
Comment (yangineer) [inline hxietest.md]: Where is Place 2 exactly? Please be more creative.

Comment (hxietest) [inline hxietest.md]: Test Comments

**Review by hxietest — COMMENTED** [review]
Test Review Summary

**Review by yangineer — COMMENTED** [review]
Looks like you haven't updated the placeholder text.

File: hxietest.md (hunk 2 above)
Comment (michellessa2) [reply hxietest.md]: @hxietest did you have Chigago style pizza when you were down there?

**Review by hxietest — COMMENTED** [review]
Comment

File: hxietest.md (hunk 2 above)
Comment (hxietest) [reply hxietest.md]: I didn't get a chance...

**Pull request merged**